agent/engine/task_manager_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build !linux,windows
+
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package engine
+
+// SetupCgroup sets up the cgroup for each managed task
+func (mtask *managedTask) SetupCgroup() error {
```

Comment:
Again, these should be in platform-specific wrappers such that we don't even compile in these symbols on Windows.